src/js/player.js

```diff
@@ -404,6 +405,9 @@ class Player extends Component {
     // Make this an evented object and use `el_` as its event bus.
     evented(this, {eventBusKey: 'el_'});
 
+    if (this.fluid_) {
+      this.on('vjs_reset', this.updateStyleEl_);
```

Comment:
so, the idea is to rerun `updateStyleEl_` when reset is called? Makes sense to me.

Reply:
so, `fluid` gets called in `createEl` so, I'm not sure we actually need this here. We can probably get away with just the piece inside the `fluid` method.

Reply:
Fluid is called first time before createEl, player is not evented yet. So no listener. Thats why the listener need to be added after createEl. I will take a look if first time fluid can move after player evented, but a delayed listener also seems a viable solution for me.

Reply:
I see, 🤔